pkgs/tools/filesystems/gcsfuse/default.nix

```diff
@@ -14,10 +14,14 @@ buildGoPackage rec {
     sha256 = "1lj9czippsgkhr8y3r7vwxgc8i952v76v1shdv10p43gsxwyyi9a";
   };
 
-  # Fully vendored, apparently.
-  goDeps = null;
+  propagatedBuildInputs = [ fuse ];
```

Comment:
Does it really use libfuse? I thought has go comes with is own implementation: https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/internal/fs/fs.go#L29